packages/schematics/update/update/npm.ts

```diff
@@ -85,6 +85,7 @@ export function getNpmPackageJson(
       return concat(
         getNpmConfigOption('proxy'),
         getNpmConfigOption('https-proxy'),
+        getNpmConfigOption('strict-ssl'),
```

Comment:
Is getNpmConfigOption('strict-ssl') provides true default when strict-ssl is missing in config?